packages/cli/test/scripts/add.test.js

```diff
@@ -78,6 +79,17 @@ contract('add script', function() {
     this.packageFile.contract('ImplV2').should.eq('ImplV2')
   })
 
+  it('should not add solidity libraries or external packages when adding all', function() {
```

Comment:
nit: `should not add solidity libraries or contracts from external packages when adding all`